Corona-Warn-App/src/main/res/layout/trace_location_privacy_card.xml

```diff
@@ -36,11 +36,11 @@
         layout="@layout/include_bullet_point"
         android:layout_width="@dimen/bullet_point_size"
         android:layout_height="@dimen/bullet_point_size"
-        app:layout_constraintBaseline_toBaselineOf="@id/first_bulletpoint_title"
```

Comment:
Is the renaming in this file intended? Or just a side effect from the refactoring of `vaccination_consent_privacy_card.xml`

Reply:
yeah, refactoring :(